CHANGELOG.md

```diff
@@ -2,6 +2,12 @@
 
 The change log describes what is "Added", "Removed", "Changed" or "Fixed" between each release.
 
+## 1.13.2
```

Comment:
ideally use "unreleased" here to avoid confusion when this is not released yet.

Reply:
:+1: